test/files/pos/t8111.scala

```diff
@@ -7,10 +7,10 @@ trait T {
     foo((u: Unit) => ma)
     foo(0, (u: Any) => ma) apply ()
 
-    // crash due to side effects on the onwer of the symbol in the 
+    // crash due to side effects on the owner of the symbol in the
     // qualifier or arguments of the application during an abandoned
-    // names/defaults transform. The code type checkes because of
-    // autp-tupling which promotes and empty parmater list to `(): Unit`
+    // names/defaults transform. The code type checks because of
+    // auto-tupling which promotes an empty parameter list to `(): Unit`
```

Comment:
I only comment on merged PRs so as not to impede progress in code comments. Otherwise it's like leaning on the horn while a boy scout tries to help an old lady through the crosswalk.